### DISCUSSION TITLE
Announcing mpv.net-next presented by the .NET community

### DISCUSSION THREAD

**Discussion by stax76** [mpvnet-player/mpv.net#405] [Show and tell]
I'm generally willing to collaborate, to emphasize this I created an org and repo:

https://github.com/mpv-net-player/mpv.net-next

**Comment by mysteryx93** [comment]
Will it be cross-platform compatible? What UI framework are you settling on?

**Comment by stax76** [comment]
Cross-platform is a main goal, but I haven't yet researched it. There are existing solutions, most important mpv itself, that's where to look at first. I believe mpv uses various different native UI framework, and mpv.net does much the same, using Win32, all render features are possible that way.
